_rules/6cbcbe.md

```diff
@@ -0,0 +1,269 @@
+---
+name: Links with identical accessible names serve equivalent purpose
+test_type: atomic
+
+description: |
+  This rule checks that links with identical accessible names serve an equivalent purpose.
+
+success_criterion: 
+- 2.4.9 # Link Purpose (Link Only)
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Anne Thyme Nørregaard
+---
+
+## Test Procedure
+
+### Applicability
+
+This rule applies to any two or more HTML or SVG elements that have the [semantic role](#semantic-role) of `link`, or a role that inherits from the `link` role, are [included in the accessibility tree](#included-in-the-accessibility-tree), and that have [matching](#matching-text-content) [accessible names](#accessible-name) that contain [text content](#text-content).
```

Comment:
What does it mean for a rule to apply to "any two or more HTML or SVG elements"?

Reply:
What if it is "a set of two or more..."?

Reply:
If I'm not mistaken, an ACT rule can only apply to individual test targets and not sets of test targets. The same goes for expectations.

Reply:
I can't see why it couldn't apply to a set... For reference: https://w3c.github.io/wcag-act/act-rules-format.html#applicability

Reply:
I do think I'm mistaken then 😄